docs/agents/fts_agent.rst

```diff
@@ -0,0 +1,64 @@
+.. highlight:: rst
+
+.. _fts_stage:
+
+=====================
+FTS Agent
+=====================
+
+This agent is used to communicate with the FTS mirror stage for two FTSs with
+Aerotech motion controllers. 
```

Comment:
I know there are other FTS in the collaboration. Is there a good way to distinguish them? Is it by motion controller? Inevitably we'll probably run into naming choices with the Agents corresponding to this and the other FTS. Is there a better, more specific name we could choose?

Reply:
That is a good point. We're sending our second FTS to UCSD and it should have the exact same controller as this one. So it should work the same but if there's small differences I'll help debug to get them both working on the same agent. As far as I know these two are the only two that could end up at the site? 

The other FTS I know about is the one being used at K2SO. I don't know if @jseibert575  has plans to make an OCS agent for that one?